uk.co.compucorp.civicrm.tasksassignments/tasksassignments.php

```diff
@@ -289,8 +289,12 @@ function tasksAssignments_civicrm_alterAngular(\Civi\Angular\Manager $angular) {
 
   $changeSet->alterHtml('~/crmCaseType/timelineTable.html', function (phpQueryObject $doc) {
     _tasksAssignments_change_add_activity_dropdown_placeholder($doc);
-    _tasksAssignments_remove_columns_from_timeline($doc);
     _tasksAssignments_change_column_text($doc);
+
+    $columnsToBeRemovedFromTimeline = [ 'Status', 'Reference', 'Select' ];
+    foreach ($columnsToBeRemovedFromTimeline as $columnName) {
+      _tasksAssignments_remove_column_from_timeline($doc, $columnName);
```

Comment:
Can we move the loop inside the `_tasksAssignments_remove_column_from_timeline` function so as to keep the hook clean and free from a lot of logic and also consistency with other helper methods in the hook? The logic should reside in the function. The function name can be pluralized like you did initially `_tasksAssignments_remove_columns_from_timeline`.